### ISSUE TITLE
Enbale tooltip on the point click in mobile webView

### ISSUE THREAD

**Issue by neswine** [highcharts/highcharts#5927]
I am using highchart in an hybrid WebApp and am having inconsistency in tooltip display. In the app, the requirement is to show the tooltip only for the clicked point. This behaviour is working for me when I render the charts in desktop or render it in a iframe even for a congested graph. But when I render the same in mobile WebView, the tooltip is not shown for the point that I clicked when the graph is congested. The tooltip is also displayed when I click anywhere in the vertical grid line.

I  have uploaded a video showing the issue.
[https://www.youtube.com/watch?v=oPR0F25HXt0](https://www.youtube.com/watch?v=oPR0F25HXt0)

#### Expected behaviour
show the tooltip only for the clicked point

#### Actual behaviour
tooltip not shown for the value clicked

#### Option used for highchart
[https://jsfiddle.net/aeddjxdt/1/](url)

#### Affected browser(s)
mobile webviews

**Comment by pawelfus** [comment]
Thank you for reporting!

Related to: #4924 (possible duplicate).

**Comment by neswine** [comment]
We added a workaround mentioned in the https://highcharts.uservoice.com/forums/55896-highcharts-javascript-api/suggestions/2607304-allow-the-tooltip-to-appear-when-a-point-is-clicke for enabling tooltip on point clicks. Although this workaround was working in desktops, it wasn't working in mobile webViews where we didn't get the tooltip on the point that we clicked. We also got another issue in tooltip rendering. The issue is that on clicking the points rapidly in mobile webView, the tooltip movement from one point to another was sluggish. We checked and saw that the click event was getting triggered multiple times which called tooltip refresh function multiple times which made the tooltip movement sluggish.

Wanted to know whether there's any update on this issue so that we could remove this workaround and add a proper fix for this issue.
